packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -331,42 +331,25 @@ export class PanelBase extends React.Component<IPanelProps, IPanelState> impleme
 
   private _onRenderNavigationContent = (props: IPanelProps): JSX.Element | null => {
     const { closeButtonAriaLabel, hasCloseButton, onRenderHeader = this._onRenderHeader } = props;
-    const theme = getTheme();
     if (hasCloseButton) {
-      // TODO -Issue #5689: Comment in once Button is converted to mergeStyles
-      // const iconButtonStyles = this._classNames.subComponentStyles
-      // ? (this._classNames.subComponentStyles.iconButton as IStyleFunctionOrObject<IButtonStyleProps, IButtonStyles>)
-      // : undefined;
+      const iconButtonStyles = this._classNames.subComponentStyles
+        ? this._classNames.subComponentStyles.closeButton()
+        : undefined;
```

Comment:
Not that important but you could use the new `?.` syntax here
```suggestion
      const iconButtonStyles = this._classNames.subComponentStyles?.closeButton();
```